tests/unit/assessment/test_clusters.py

```diff
@@ -19,87 +19,10 @@
-            },
-            spark_context_id=5134472582179565315,
-            spark_env_vars=None,
-            spark_version="13.3.x-cpu-ml-scala2.12",
-            cluster_id="0915-190044-3dqy6751",
-        ),
-        ClusterDetails(
-            cluster_name="Tech Summit FY24 Cluster-1",
-            autoscale=AutoScale(min_workers=1, max_workers=6),
-            spark_context_id=5134472582179565315,
-            spark_env_vars=None,
-            policy_id="D96308F1BF0003A7",
-            spark_version="13.3.x-cpu-ml-scala2.12",
-            cluster_id="0915-190044-3dqy6751",
-        ),
-    ]
-
-    ws = Mock()
-
-    ws.cluster_policies.get().definition = (
-        '{\n  "spark_conf.fs.azure.account.auth.type": {\n    '
-        '"type": "fixed",\n    "value": "OAuth",\n   '
-        ' "hidden": true\n  },\n  "spark_conf.fs.azure.account.oauth.provider.type": {\n   '
-        ' "type": "fixed",\n    "value": '
-        '"org.apache.hadoop.fs.azurebfs.oauth2.ClientCredsTokenProvider",\n    '
-        '"hidden": true\n  },\n  "spark_conf.fs.azure.account.oauth2.client.id": {\n    '
-        '"type": "fixed",\n    "value": "fsfsfsfsffsfsf",\n    "hidden": true\n  },\n  '
-        '"spark_conf.fs.azure.account.oauth2.client.secret": {\n    "type": "fixed",\n    '
-        '"value": "gfgfgfgfggfggfgfdds",\n    "hidden": true\n  },\n  '
-        '"spark_conf.fs.azure.account.oauth2.client.endpoint": {\n    '
-        '"type": "fixed",\n    '
-        '"value": "https://login.microsoftonline.com/1234ededed/oauth2/token",\n    '
-        '"hidden": true\n  }\n}'
-    )
-    ws.cluster_policies.get().policy_family_definition_overrides = (
-        '{\n  "spark_conf.fs.azure.account.auth.type": {\n    '
-        '"type": "fixed",\n    "value": "OAuth",\n   '
-        ' "hidden": true\n  },\n  "spark_conf.fs.azure.account.oauth.provider.type": {\n   '
-        ' "type": "fixed",\n    "value": '
-        '"org.apache.hadoop.fs.azurebfs.oauth2.ClientCredsTokenProvider",\n    '
-        '"hidden": true\n  },\n  "spark_conf.fs.azure.account.oauth2.client.id": {\n    '
-        '"type": "fixed",\n    "value": "fsfsfsfsffsfsf",\n    "hidden": true\n  },\n  '
-        '"spark_conf.fs.azure.account.oauth2.client.secret": {\n    "type": "fixed",\n    '
-        '"value": "gfgfgfgfggfggfgfdds",\n    "hidden": true\n  },\n  '
-        '"spark_conf.fs.azure.account.oauth2.client.endpoint": {\n    '
-        '"type": "fixed",\n    '
-        '"value": "https://login.microsoftonline.com/1234ededed/oauth2/token",\n    '
-        '"hidden": true\n  }\n}'
-    )
-
-    crawler = ClustersCrawler(ws, MockBackend(), "ucx")._assess_clusters(sample_clusters)
-    result_set = list(crawler)
+def test_cluster_assessment():
+    ws = workspace_client_mock(clusters="assortment-conf.json")
+    crawler = ClustersCrawler(ws, MockBackend(), "ucx")
+    result_set = list(crawler.snapshot())
 
     assert len(result_set) == 4
```

Comment:
Can you do assertions on failure messages received? That way we'll be confident in tests doing what they should